### DISCUSSION TITLE
iOS crash

### DISCUSSION THREAD

**Discussion by coreyfu1** [Tencent/libpag#1865] [General] [answered]
使用libpag的过程中，发现几个比较频繁的崩溃，版本4.3.27
调试模式下：
<img width="1376" alt="截屏2023-11-13 13 27 54" src="https://github.com/Tencent/libpag/assets/17798137/0cd9aee3-474c-4a35-ae2f-14dce66287e8">
线上：
<img width="1299" alt="截屏2023-11-13 13 35 11" src="https://github.com/Tencent/libpag/assets/17798137/76843724-619e-4730-afe8-0f28da326660">
<img width="844" alt="截屏2023-11-13 13 35 39" src="https://github.com/Tencent/libpag/assets/17798137/9bc9814a-a80d-407d-9fe5-e36606779cdf">

**Comment by domchen** [comment, marked as answer]
这个是无效的堆栈，符号还原错了。检查一下是否来自免费版的Bugly，是的话尝试手动还原堆栈信息。另外也确认一下还原用的符号文件与sdk版本是否匹配。
